### ISSUE TITLE
Prism Latest version problem

### ISSUE THREAD

**Issue by Coolkeke** [AvaloniaCommunity/Prism.Avalonia#43]
When is Prism planning to release 8.1.97.3-preview.11.5? Anvaloia has released 8.1.97.3-preview.11.5. I think you have such a piece of code in your latest project
protected abstract AvaloniaObject CreateShell();
Currently, it is still used in 8.1.97.3-preview.11.4
protected abstract IAvaloniaObject CreateShell();
Such a method, but in the latest Avalonia 8.1.97.3 - preview. 11.5, there is no such interface as IAvaloniaObject. How do I deal with this problem?

Here is my project Demo, please help me to have a look
[Todo.zip](https://github.com/AvaloniaCommunity/Prism.Avalonia/files/10610472/Todo.zip)

**Comment by DamianSuess** [comment]
@Coolkeke thank you for reaching out. I created a post in the Discussion forum, #44 [Avalonia v11 Preview 5 - Project Status](https://github.com/AvaloniaCommunity/Prism.Avalonia/discussions/44)

Yes, this is unfortunate when migrating from v11's Preview 4 to 5, as noted by both PR #39 and Avalonia's Wiki page titled, [Breaking Changes](https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes). 

Considering there are [1,422 Commits and 1,947 File changes](https://github.com/AvaloniaUI/Avalonia/compare/release/11.0.0-preview4...release/11.0.0-preview5), I'm doing my best to swiftly upgrade and document the changes required over here in the [Upgrading-to-Avalonia-11.md](https://github.com/AvaloniaCommunity/Prism.Avalonia/blob/develop/Upgrading-to-Avalonia-11.md) file.

In short, Prism.Avalonia only supports up to Avalonia v11-Preview 4 for now.

**Comment by Coolkeke** [comment]
@DamianSuess  Thank you for your reply. I have publicized your project in the company. Many projects in our company are using the open source tools provided by your team. We also appreciate your Avalonia. Prism project. This is a very good project. We look forward to your latest news. Thank you again for your reply!!

**Comment by DamianSuess** [comment]
You're welcome @Coolkeke and thank you for the support. Best wishes on your projects.

Keep an eye out for the new update coming soon

**Comment by DamianSuess** [comment]
@Coolkeke just to let you know, after resolving PR #47 I will publish the latest NuGet.

**Comment by Coolkeke** [comment]
Ok, thank you for reminding

**Comment by DamianSuess** [comment]
Good morning @Coolkeke just wanted to let you know that the library has been published as per announcement #48.  Please keep in mind, this package is still in preview which warns of instability.

For what it's worth, at my office we are still using Prism.Avalonia v8.1.97.2 for Avalonia v0.10.18 for production application and sandboxing the v11 pre-releases until its changes become fully stable.

Thank you again for your support!
- Damian
